include/pros/ext_adi.h

```diff
@@ -687,6 +687,239 @@ ext_adi_potentiometer_t ext_adi_potentiometer_init(uint8_t smart_port, uint8_t a
  */
 double ext_adi_potentiometer_get_angle(ext_adi_potentiometer_t potentiometer);
 
+/**
+ * Reference type for an initialized addressable led.
+ *
+ * This merely contains the port number for the led, unlike its use as an
+ * object to store led data in PROS 2.
```

Comment:
could you change any references to PROS 2 to "the C++ API."
```suggestion
 * object to store led data in the C++ API.
```